src/cmds/get/index.ts

```diff
@@ -1,12 +1,14 @@
 import addGetABICmd from './abi'
 import addGetGroupCmd from './group'
+import addGroupEventsCmd from './events'
```

Comment:
let's rename this to `addGetEventsCmd` to keep parity with how it relates to what the command looks like when typed in the CLI